userspace/libsinsp/parsers.cpp

```diff
@@ -3402,3 +3405,16 @@ void sinsp_parser::parse_container_evt(sinsp_evt *evt)
 
 	m_inspector->m_container_manager.add_container(container_info);
 }
+
+void sinsp_parser::parse_semop_exit(sinsp_evt *evt)
```

Comment:
Parsers are needed to update `libsisnsp` state (like when there are new processes, new thread, open files etc). In this case you are not updating it, why are you adding this code?